### PR TITLE
FSAR-32: Temporary Message.

### PR DESCRIPTION
**Pull request by beloglazov91**
Please provide a thorough description of your component and its usage with screenshots.

## Checklist before opening a PR

Before opening a PR, please make sure:

- [ ] your work is in a branch based on and synced with the develop branch.
- [ ] your scss and twig files are imported in the javascript file. Without this, Webpack won't be able to compile them.
- [ ] your newComponent is imported in the index.js file.
- [ ] your folder hierarchy follows the proper structure.
- [ ] your code follows the coding standards and naming conventions defined.
- [ ] your code has been tested as AA accessibility compliant to the WCAG 2.1 AA standard.
- [ ] your code has been tested in all the browsers listed in the GDS browser's list, including IE11.
- [ ] your code has been tested in all the different screensizes defined.

### CODE REVIEW

File: src/components/general/TemporaryMessage/temporaryMessage.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
+@import "src/lib.scss";
+
+.temporary-message {
+  background-color: var(--fsa__grey-hover);
+  border: 1px solid var(--fsa_border);
+  font-size: 0.8rem;
+  padding: 1rem 0.88rem;
```
Comment (c-ancia) [inline src/components/general/TemporaryMessage/temporaryMessage.scss:7]: 0.88rem is a weird number, what does it match please? I think Rachael tried to keep these numbers as round as possible.

File: src/components/general/TemporaryMessage/temporaryMessage.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  position: relative;
+
+  @media screen and (min-width: $fsa-md) {
+    padding: 1rem 2rem;
+  }
+
+  @media screen and (min-width: $fsa-lg) {
+    font-size: 0.9rem;
+    padding: 1rem 18.10rem;
```
Comment (c-ancia) [inline src/components/general/TemporaryMessage/temporaryMessage.scss:16]: also here, 18.10rem is a weird number, what does it match please?

File: src/components/general/TemporaryMessage/temporaryMessage.scss [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      &:before {
+        background-image: url("./assets/warning-white.svg");
+      }
+    }
+  }
+
+  &__content {
+    display: flex;
+    justify-content: flex-start;
+    padding-left: 0.44rem;
```
Comment (c-ancia) [inline src/components/general/TemporaryMessage/temporaryMessage.scss:37]: can you also tell me 0.44rem please?

File: src/components/general/TemporaryMessage/temporaryMessage.scss [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+    @media screen and (min-width: $fsa-md) {
+      padding-left: 0.84rem;
```
Comment (c-ancia) [inline src/components/general/TemporaryMessage/temporaryMessage.scss:40]: and 0.84rem?

**Comment by beloglazov91** [comment]
@c-ancia I've fixed all problems.

**Pull request merged**